src/style.css

```diff
@@ -1 +1,73 @@
 /* Styles go here */
+
+.lessonName {
+  display: none;
+}
+
+.body {
+  margin: 0;
+}
+
+.header {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+  align-self: center;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  list-style: none;
+}
+
+.head__List {
+  margin: 0 10px;
+}
+
+.links {
+  position: relative;
+  display: flex;
+  height: 60px;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #000;
+  align-items: center;
+}
+
+.links:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  margin-left: 0;
+  color: #00acdc;
+}
+
+.start {
+  margin-left: 0;
+}
+
+.end {
+  margin-right: 0;
+}
```

Comment:
don't use such classes for setting styles for n-th element. you have pseudoclasses for it. You can use `:first-child` and `:last-child` here